internal/server/data/group.go

```diff
@@ -71,59 +93,49 @@ func groupIDsForUser(tx ReadTxn, userID uid.ID) ([]uid.ID, error) {
 	return result, rows.Err()
 }
 
-func DeleteGroups(db GormTxn, selectors ...SelectorFunc) error {
-	toDelete, err := ListGroups(db, nil, selectors...)
+func DeleteGroup(tx WriteTxn, id uid.ID) error {
```

Comment:
This was only ever called with a single group ID, so this function has changed to only accept that one ID, which removes the need to list groups.

Group membership also changed from using `RemoveUsersFromGroup` to a query that deletes all rows that match the group ID.